### ISSUE TITLE
Add ingredient to cosmetic surgeries

### ISSUE THREAD

**Issue by kaptain-kavern** [kaptain-kavern/DESurgeries#1]
- **Cosmetic Surgeries** : Like steel or wood maybe?
- **Repair Finger or Toe** : Wood for splint? (enforce creation of splint items?)

**Comment by BlueWinds** [comment]
My headcanon was always that these were advanced medical procedures, regrowing the damaged tissues - or at the very least providing quite advanced prosthesis. So my preference would by no splint items, to continue letting my pretend it's not just a wooden nose.

Edit - I hope you don't mind me commenting on your issues here, which look like they're more for your benefit than anyone else's. :)

**Comment by kaptain-kavern** [comment]
No no I don't mind. Please contribute :-D in fact that was the reason I wrote this.

I was working more on ADogSaid lately... 

Yep after some thinking ( and readings your suggestions) I think it's better to let things like they are. 

That way this mod will be a sort of extended medical mod only. And for other options there is the excellent EPOE mod. I have compatibility with it almost ready, I just wait for Ykara (the mod creator) to say yes

**Comment by BlueWinds** [comment]
I don't think you need to wait for permission to release a compatibility patch. You're not using their code, claiming credit, or doing anything else questionable. I mean, it's nice to notify, but I don't think you need _permission_.

**Comment by kaptain-kavern** [comment]
Alpha phase for the EPOE compatibility patch : https://github.com/kaptain-kavern/DESurgeries/commit/be6d2b20f058bb29ebb1ed52d9a66f2703df75de
i just need to massively dev mode test this version
